flask_appbuilder/security/manager.py

```diff
@@ -539,7 +576,7 @@ def get_oauth_user_info(self, provider, resp):
         # JWT token needs to be base64 decoded.
         # https://docs.microsoft.com/en-us/azure/active-directory/develop/
         # active-directory-protocols-oauth-code
-        if provider == "azure":
+        elif provider == "azure":
```

Comment:
This dict is missing the groups claim returned from azure if you configure the enterprise app to return group information. The key name is `groups`, and it's configurable in azure, but I think it's sensible to just require the default - 99% of cases won't change that.

So just adding

```python
"groups": me.get("groups", [])
```` 

to the dict below would work. But documentation should also be updated to instruct users to configure the group claim to be sent with the token for the app, because it's optional, so not sent by default (that's why I had the `[]` there). @thesuperzapper can you add that pls?